site/.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
I don't think we need this. which tool will use this file?

Reply:
it is my local ruby "rbenv", will drop it sorry